invenio_app_ils/ui/public/index.html

```diff
@@ -28,7 +28,7 @@
     <noscript> You need to enable JavaScript to run this app. </noscript>
     <div
       id="invenio-config"
-      data-config="{&quot;items&quot;: {&quot;loanActiveStates&quot;:[&quot;ITEM_AT_DESK&quot;, &quot;ITEM_ON_LOAN&quot;, &quot;ITEM_IN_TRANSIT_FOR_PICKUP&quot;, &quot;ITEM_IN_TRANSIT_TO_HOUSE&quot;],&quot;search&quot;: {&quot;aggs&quot;: [&quot;status&quot;, &quot;medium&quot;, &quot;circulation_status&quot;], &quot;sortBy&quot;: {&quot;onEmptyQuery&quot;: &quot;mostrecent&quot;, &quot;values&quot;: [{&quot;field&quot;: &quot;mostrecent&quot;, &quot;order&quot;: 1, &quot;title&quot;: &quot;Newest&quot;}, {&quot;field&quot;: &quot;bestmatch&quot;, &quot;order&quot;: 2, &quot;title&quot;: &quot;Best match&quot;}]}, &quot;sortOrder&quot;: [&quot;asc&quot;, &quot;desc&quot;]}}, &quot;loans&quot;: {&quot;search&quot;: {&quot;aggs&quot;: [&quot;state&quot;], &quot;sortBy&quot;: {&quot;onEmptyQuery&quot;: &quot;mostrecent&quot;, &quot;values&quot;: [{&quot;field&quot;: &quot;mostrecent&quot;, &quot;order&quot;: 1, &quot;title&quot;: &quot;Newest&quot;}, {&quot;field&quot;: &quot;bestmatch&quot;, &quot;order&quot;: 2, &quot;title&quot;: &quot;Best match&quot;}]}, &quot;sortOrder&quot;: [&quot;asc&quot;, &quot;desc&quot;]}}}"
+      data-config="{&quot;items&quot;: {&quot;available&quot;: {&quot;status&quot;: &quot;LOANABLE&quot;}, &quot;loanActiveStates&quot;: [&quot;ITEM_AT_DESK&quot;, &quot;ITEM_ON_LOAN&quot;, &quot;ITEM_IN_TRANSIT_FOR_PICKUP&quot;, &quot;ITEM_IN_TRANSIT_TO_HOUSE&quot;], &quot;search&quot;: {&quot;aggs&quot;: [&quot;status&quot;, &quot;medium&quot;, &quot;circulation_status&quot;], &quot;sortBy&quot;: {&quot;onEmptyQuery&quot;: &quot;mostrecent&quot;, &quot;values&quot;: [{&quot;field&quot;: &quot;mostrecent&quot;, &quot;order&quot;: 1, &quot;title&quot;: &quot;Newest&quot;}, {&quot;field&quot;: &quot;bestmatch&quot;, &quot;order&quot;: 2, &quot;title&quot;: &quot;Best match&quot;}]}, &quot;sortOrder&quot;: [&quot;asc&quot;, &quot;desc&quot;]}}, &quot;loans&quot;: {&quot;search&quot;: {&quot;aggs&quot;: [&quot;state&quot;], &quot;sortBy&quot;: {&quot;onEmptyQuery&quot;: &quot;mostrecent&quot;, &quot;values&quot;: [{&quot;field&quot;: &quot;mostrecent&quot;, &quot;order&quot;: 1, &quot;title&quot;: &quot;Newest&quot;}, {&quot;field&quot;: &quot;bestmatch&quot;, &quot;order&quot;: 2, &quot;title&quot;: &quot;Best match&quot;}]}, &quot;sortOrder&quot;: [&quot;asc&quot;, &quot;desc&quot;]}}}"
```

Comment:
@kprzerwa or @zzacharo could you please maybe change in the **next** PR the "namespacing" of this?
It was my mistake... WDYT if we stick to circulation for things around loans? It is to avoid confusion...

So in this case, something like:
`config.circulation.loan_active_states`
or similar